ui/lib/pki/addon/components/parsed-certificate-info-rows.ts

```diff
@@ -13,7 +13,30 @@ import { parsedParameterKeys } from 'vault/utils/parse-pki-cert-oids';
  *
  * @param {object} model - object of parsed attributes from parse-pki-cert util
  */
-export default class ParsedCertificateInfoRowsComponent extends Component {
+
+interface AttrsByKeys {
+  other_sans: { label: string };
+  alt_names: { label: string };
+  uri_sans: { label: string };
+  ip_sans: { label: string };
+  permitted_dns_domains: { label: string };
+  exclude_cn_from_sans: { label: string };
+  use_pss: { label: string };
+  ttl: { label: string };
+  ou: { label: string };
+  not_valid_after: { formatDate: string };
+  not_valid_before: { formatDate: string };
+}
+
+interface Args {
+  model: {
+    parsing_errors: {
+      message: string;
+    }[];
+  };
```

Comment:
Mostly as an FYI, `parsing_errors` is an array of error objects. You might be able to import the ParsedCertificateData interface from [here](https://github.com/hashicorp/vault/blob/9de1cfcf8279c07135b4d8ad06511b0590cb47e1/ui/types/vault/utils/parse-pki-cert.d.ts#L7) (which would need `export` added before the interface)
and do something like 
```
 import { ParsedCertificateData } from 'vault/vault/utils/parse-pki-cert';
```
```suggestion
  model: ParsedCertificateData
  };
```

I'm just not sure if typescript will be satisfied with `<Error>` for the `message` key 🤔

Reply:
ahh good call! I didn't think of that :)